ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -63,6 +64,18 @@ def _pcs_cluster_start(self):
             Log.info(f"Pcs cluster start retry index : {retry_index}")
         return _status
 
+    @staticmethod
+    def load_json_file(json_file):
```

Comment:
Move this to parent class

Reply:
Done

Reply:
move load json to pcs_controller not cluster controller
and rename load_json_file to _load_json_file

Reply:
Done